src/rdb_protocol/changefeed.cc

```diff
@@ -1572,8 +1570,8 @@ class flat_sub_t : public subscription_t {
             queue->add(change_val_t(
                 std::make_pair(shard_uuid, stamp),
                 pkey,
-                old_val,
-                new_val
+                std::move(old_val),
+                std::move(new_val)
```

Comment:
This is now fixed -- the type is noncopyable now.  (And I think it was actually being moved before?)